scripts/devenv.sh

```diff
@@ -9,6 +9,7 @@ docker run -it \
 	--privileged \
 	-v /var/run/docker.sock:/var/run/docker.sock \
 	-v `pwd`:/gopath/src/github.com/Azure/acs-engine \
+	-v ~/.azure:/root/.azure \
```

Comment:
Noticed if file missing, docker run -v will create it for u.
Could you add something before running this docker run, something like `touch ~/.azure`. It will also help to resolve the issue which Cole pointed out, docker created the file with root permission.